src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -90,7 +90,7 @@ Author(s):
     X(hstring, TabTitle, "tabTitle")                                                                                                                           \
     X(Model::BellStyle, BellStyle, "bellStyle", BellStyle::Audible)                                                                                            \
     X(IEnvironmentVariableMap, EnvironmentVariables, "environment", nullptr)                                                                                   \
-    X(bool, UseAtlasEngine, "useAtlasEngine", Feature_AtlasEngine::IsEnabled())                                                                                \
+    X(bool, UseAtlasEngine, "useAtlasEngine", true)                                                                                                            \

```

Comment:
I think for this period we'll want to change the _name_ of the setting, so that people who opted out when it was broken get another chance to test it out. I was wondering if we should do that during Preview, though... 🤔

Reply:
Oh... We could check the `AtlasEngine_Usage` metric to see how many users opted out. This would tell us both, how many people this switch might (negatively) surprise and also whether introducing a new name for this setting is useful in the first place. Should we do that?

Reply:
> might (negatively) surprise

To be fair, #16278 isn't going to do those people any favors 😆

Reply:
I need to regain access to Kusto to tell you about Atlas usage

Reply:
> To be fair, https://github.com/microsoft/terminal/pull/16278 isn't going to do those people any favors 😆

That's true, but it targets the 1.20 branch, so if there's anything in AtlasEngine that upsets an unreasonable amount of people, then I'll fix it ASAP before 1.20 is released as Preview.